main.py

```diff
@@ -1,7 +1,37 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+
+    # create
+    genres = ["Western", "Action", "Dramma"]
+    actors = [
+        ("George", "Klooney"), ("Kianu", "Reaves"), ("Scarlett", "Keegan"),
+        ("Willz", "Smith"), ("Jaden", "Smith"), ("Scarlett", "Johansson")
```

Comment:
There is a typo in the actor's name 'Willz Smith'. It should be 'Will Smith' to ensure consistency and correctness.